aboutMe.js

```diff
@@ -0,0 +1,32 @@
+//Chaning fonts:
+document.querySelector("body").style.fontFamily="Arial, sans-serif";
+//Changing attributes:
+let nickname = document.querySelector("#nickname");
+let favorite = document.querySelector("#favorite");
+let hometown = document.querySelector("#hometown");
+
+//replacing each span with my information
+nickname.innerText = "Cha";
+favorite.innerText = "Dogs";
+hometown.innerText = "Madina";
+
+//selecting all li
+let list = document.querySelectorAll("li");
+
+//for loop to change all li
+for (let i=0; i<document.querySelectorAll("li"); i++){

```

Comment:
I think you forget to add .length to your array
```suggestion
for (let i=0; i<document.querySelectorAll("li").length; i++){
```